gk/main.c

```diff
@@ -1383,6 +1385,16 @@ process_pkts_front(uint16_t port_front, uint16_t port_back,
 			continue;
 		}
 
+		if (packet.flow.proto == ETHER_TYPE_IPv4 &&
+				 !ipv4_configured_front) {
+			drop_packet_front(pkt, instance);
```

Comment:
This ``if`` statement for IPv4 packets and the below statement for IPv6 packets are the same, so we can combine the conditions check to one, and the code becomes simpler.

The same comment applies to the back interface.

Reply:
Once we have a single `if` statement, please wrap the test with `unlikely()`.